### PR TITLE
[Auto] Localization - Translated Strings

### PR DESCRIPTION
**Pull request by csigs**

### CODE REVIEW

File: Extension/i18n/csy/walkthrough/installcompiler/install-compiler-windows.md.i18n.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
 	"walkthrough.windows.note1": "Poznámka",
 	"walkthrough.windows.note1.text": "Můžete použít sadu nástrojů C++ z Visual Studio Build Tools spolu s Visual Studio Code ke kompilaci, sestavení a ověření jakékoli kódové báze C++, pokud máte také platnou licenci Visual Studio (buď Community, Pro nebo Enterprise), kterou aktivně používáte k vývoji kódové báze C++.",
 	"walkthrough.windows.open.command.prompt": "Otevřete {0} zadáním příkazu „developer“ do nabídky Start systému Windows.",
-	"walkthrough.windows.command.prompt.name1": "Developer Command Prompt for VS",
+	"walkthrough.windows.command.prompt.name1": "Developer Command Prompt pro VS",
```
Comment (bobbrow) [inline Extension/i18n/csy/walkthrough/installcompiler/install-compiler-windows.md.i18n.json]: I opened a bug against the shortcut names a bit ago and it looks like they will become "brand" names which means they won't be localized (they're currently not anyway). When that is official, we'll need to get loc and doc updates, but for now, maybe we reject this change here.

File: Extension/i18n/csy/walkthrough/installcompiler/install-compiler-windows10.md.i18n.json [hunk 2]
```diff
@@ -11,7 +11,7 @@
 	"walkthrough.windows.note1.text": "Můžete použít sadu nástrojů C++ z Visual Studio Build Tools spolu s Visual Studio Code ke kompilaci, sestavení a ověření jakékoli kódové báze C++, pokud máte také platnou licenci Visual Studio (buď Community, Pro nebo Enterprise), kterou aktivně používáte k vývoji kódové báze C++.",
 	"walkthrough.windows.verify.compiler": "Ověřování instalace kompilátoru",
 	"walkthrough.windows.open.command.prompt": "Otevřete {0} zadáním příkazu „developer“ do nabídky Start systému Windows.",
-	"walkthrough.windows.command.prompt.name1": "Developer Command Prompt for VS",
+	"walkthrough.windows.command.prompt.name1": "Developer Command Prompt pro VS",
```
Comment (bobbrow) [inline Extension/i18n/csy/walkthrough/installcompiler/install-compiler-windows10.md.i18n.json]: Same for this one (and any others like it)

File: Extension/i18n/deu/src/Debugger/ParsedEnvironmentFile.i18n.json [hunk 3]
```diff
@@ -4,5 +4,5 @@
  *--------------------------------------------------------------------------------------------*/
 // Do not edit this file. It is machine generated.
 {
-	"ignoring.lines.in.envfile": "Nicht analysierbare Zeilen in {0} {1} werden ignoriert: "
-}
+	"ignoring.lines.in.envfile": "Nicht analysierbare Zeilen in {0} {1}werden ignoriert: "
```
Comment (bobbrow) [inline Extension/i18n/deu/src/Debugger/ParsedEnvironmentFile.i18n.json]: add back the space?

Comment (bobbrow) [reply Extension/i18n/deu/src/Debugger/ParsedEnvironmentFile.i18n.json]: I think something's wrong with this one in general though. It ends with a `:` and a space like it expects the `{1}` at the end.

Comment (Colengms) [reply Extension/i18n/deu/src/Debugger/ParsedEnvironmentFile.i18n.json]: I think that is as expected.  All the other strings of this ID are formatted the same way (": ").  Code outputs one or more lines of text after this label.

**Review by bobbrow — APPROVED** [review]

**Pull request merged**